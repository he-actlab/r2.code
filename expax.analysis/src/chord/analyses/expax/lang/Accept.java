package chord.analyses.expax.lang;

public class Accept {
	public static boolean accept(boolean b){return b;}
	public static float accept(float f){return f;}
	public static int accept(int i){return i;}
	public static double accept(double d){return d;}
	public static byte accept(byte b){return b;}
	public static char accept(char c){return c;}
	public static short accept(short s){return s;}
	public static long accept(long l){return l;}
	
	//TODO generate enough possible accept_all
	public static int[] accept_all_FIELD2_TAG1(int[] arr) {return arr;}
	public static int[] accept_all_FIELD1_TAG11(int[] arr) {return arr;}
	public static int[] accept_all_FIELD1_TAG2(int[] arr) {return arr;}
	public static int[] accept_all_FIELD1_TAG5(int[] arr) {return arr;}
	public static float[] accept_all_FIELD1_TAG6(float[] arr) {return arr;}
	public static double[] accept_all_FIELD1_TAG1(double[] arr){return arr;}
}
