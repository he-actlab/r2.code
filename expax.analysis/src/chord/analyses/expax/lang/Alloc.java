package chord.analyses.expax.lang;

public class Alloc {
	public static void alloc_TAG1(){}
	public static void alloc_TAG2(){}
	public static void alloc_TAG3(){}
	public static void alloc_TAG4(){}
	public static void alloc_TAG5(){}
	public static void alloc_TAG6(){}
	public static void alloc_TAG7(){}
	public static void alloc_TAG8(){}
	public static void alloc_TAG9(){}
	public static void alloc_TAG10(){}
	public static void alloc_TAG11(){}
	public static void alloc_TAG12(){}
	public static void alloc_TAG13(){}
	public static void alloc_TAG14(){}
	public static void alloc_TAG15(){}
	public static void alloc_TAG16(){}
	public static void alloc_TAG17(){}
	public static void alloc_TAG18(){}
	public static void alloc_TAG19(){}
	public static void alloc_TAG20(){}
}
