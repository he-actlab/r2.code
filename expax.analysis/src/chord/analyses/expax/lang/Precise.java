package chord.analyses.expax.lang;

public class Precise {
	public static float precise(float f){return f;}
	public static int precise(int i){return i;}
	public static double precise(double d){return d;}
	public static byte precise(byte b){return b;}
	public static char precise(char c){return c;}
	public static short precise(short s){return s;}
	public static long precise(long l){return l;}
	
	//TODO generate enough possible precise_all
	public static double[] precise_all_FIELD1_TAG1(double[] d){return d;}
	public static int[] precise_all_FIELD2_TAG1(int[] arr) {return arr;}
}
