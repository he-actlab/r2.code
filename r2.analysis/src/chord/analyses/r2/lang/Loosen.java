package chord.analyses.r2.lang;

public class Loosen {
	public static boolean loosen(boolean b){return b;}
	public static float loosen(float f){return f;}
	public static int loosen(int i){return i;}
	public static double loosen(double d){return d;}
	public static byte loosen(byte b){return b;}
	public static char loosen(char c){return c;}
	public static short loosen(short s){return s;}
	public static long loosen(long l){return l;}
	
	public static boolean loosen_invasive(boolean b){return b;}
	public static float loosen_invasive(float f){return f;}
	public static int loosen_invasive(int i){return i;}
	public static double loosen_invasive(double d){return d;}
	public static byte loosen_invasive(byte b){return b;}
	public static char loosen_invasive(char c){return c;}
	public static short loosen_invasive(short s){return s;}
	public static long loosen_invasive(long l){return l;}
	
	//TODO generate enough possible loosen_all
	public static int[] loosen_all_FIELD2_TAG1(int[] arr) {return arr;}
	public static int[] loosen_all_FIELD1_TAG11(int[] arr) {return arr;}
	public static int[] loosen_all_FIELD1_TAG2(int[] arr) {return arr;}
	public static int[] loosen_all_FIELD1_TAG5(int[] arr) {return arr;}
	public static float[] loosen_all_FIELD1_TAG6(float[] arr) {return arr;}
	public static double[] loosen_all_FIELD1_TAG1(double[] arr){return arr;}
}
