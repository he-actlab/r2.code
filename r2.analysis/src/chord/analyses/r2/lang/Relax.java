package chord.analyses.r2.lang;

public class Relax {
	public static boolean relax(boolean b){return b;}
	public static float relax(float f){return f;}
	public static int relax(int i){return i;}
	public static double relax(double d){return d;}
	public static byte relax(byte b){return b;}
	public static char relax(char c){return c;}
	public static short relax(short s){return s;}
	public static long relax(long l){return l;}
	
	//TODO generate enough possible relax_all
	public static int[] relax_all_FIELD2_TAG1(int[] arr) {return arr;}
	public static int[] relax_all_FIELD1_TAG11(int[] arr) {return arr;}
	public static int[] relax_all_FIELD1_TAG2(int[] arr) {return arr;}
	public static int[] relax_all_FIELD1_TAG5(int[] arr) {return arr;}
	public static float[] relax_all_FIELD1_TAG6(float[] arr) {return arr;}
	public static double[] relax_all_FIELD1_TAG1(double[] arr){return arr;}
}
