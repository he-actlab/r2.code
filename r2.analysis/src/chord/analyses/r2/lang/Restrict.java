package chord.analyses.r2.lang;

public class Restrict {
	public static float restrict(float f){return f;}
	public static int restrict(int i){return i;}
	public static double restrict(double d){return d;}
	public static byte restrict(byte b){return b;}
	public static char restrict(char c){return c;}
	public static short restrict(short s){return s;}
	public static long restrict(long l){return l;}
	public static boolean restrict(boolean b){return b;}
	
	//TODO generate enough possible precise_all
	public static double[] restrict_all_FIELD1_TAG1(double[] d){return d;}
	public static int[] restrict_all_FIELD2_TAG1(int[] arr) {return arr;}
}
