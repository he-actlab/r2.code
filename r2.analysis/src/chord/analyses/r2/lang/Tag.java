package chord.analyses.r2.lang;

public class Tag {
	public static void TAG1(){}
	public static void TAG2(){}
	public static void TAG3(){}
	public static void TAG4(){}
	public static void TAG5(){}
	public static void TAG6(){}
	public static void TAG7(){}
	public static void TAG8(){}
	public static void TAG9(){}
	public static void TAG10(){}
	public static void TAG11(){}
	public static void TAG12(){}
	public static void TAG13(){}
	public static void TAG14(){}
	public static void TAG15(){}
	public static void TAG16(){}
	public static void TAG17(){}
	public static void TAG18(){}
	public static void TAG19(){}
	public static void TAG20(){}
}
