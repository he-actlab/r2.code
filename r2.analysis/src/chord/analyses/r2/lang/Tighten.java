package chord.analyses.r2.lang;

public class Tighten {
	public static float tighten(float f){return f;}
	public static int tighten(int i){return i;}
	public static double tighten(double d){return d;}
	public static byte tighten(byte b){return b;}
	public static char tighten(char c){return c;}
	public static short tighten(short s){return s;}
	public static long tighten(long l){return l;}
	public static boolean tighten(boolean b){return b;}
	
	//TODO generate enough possible precise_all
	public static double[] tighten_all_FIELD1_TAG1(double[] d){return d;}
	public static int[] tighten_all_FIELD2_TAG1(int[] arr) {return arr;}
}
