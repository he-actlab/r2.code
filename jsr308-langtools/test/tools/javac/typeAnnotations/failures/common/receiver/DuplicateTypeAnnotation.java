/*
 * @test /nodynamiccopyright/
 * @bug 6843077
 * @summary check for duplicate annotations in receiver
 * @author Mahmood Ali
 * @compile/fail/ref=DuplicateTypeAnnotation.out -XDrawDiagnostics -source 1.8 DuplicateTypeAnnotation.java
 */
import java.lang.annotation.*;
class DuplicateTypeAnnotation {
  void test(@A @A DuplicateTypeAnnotation this) { }
}

@Target({ElementType.TYPE_USE, ElementType.TYPE_PARAMETER})
@interface A { }
