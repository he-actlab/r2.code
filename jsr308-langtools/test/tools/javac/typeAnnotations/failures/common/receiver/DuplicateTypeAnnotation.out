DuplicateTypeAnnotation.java:10:16: compiler.err.duplicate.annotation
1 error
